rust/azure_iot_operations_services/src/azure_device_registry/device_name_gen/adr_base_service.rs

```diff
@@ -0,0 +1,188 @@
+/* Code generated by Azure.Iot.Operations.ProtocolCompiler v0.10.0.0; DO NOT EDIT. */
+
+mod asset;
+mod asset_config_status_schema;
+mod asset_dataset_data_point_schema_element_schema;
+mod asset_dataset_destination_schema_element_schema;
+mod asset_dataset_event_stream_status;
+mod asset_dataset_schema_element_schema;
+mod asset_event_data_point_schema_element_schema;
+mod asset_event_destination_schema_element_schema;
+mod asset_event_schema_element_schema;
+mod asset_management_group_action_schema_element_schema;
+mod asset_management_group_action_status_schema_element_schema;
+mod asset_management_group_action_type_schema;
+mod asset_management_group_schema_element_schema;
+mod asset_management_group_status_schema_element_schema;
+mod asset_specification_schema;
+mod asset_status;
+mod asset_stream_destination_schema_element_schema;
+mod asset_stream_schema_element_schema;
+mod asset_update_event_schema;
+mod asset_update_event_telemetry;
+mod asset_update_event_telemetry_receiver;
+mod asset_update_event_telemetry_serialization;
+mod authentication_schema;
+mod config_error;
+mod create_detected_asset_command_invoker;
+mod create_detected_asset_request_payload;
+mod create_detected_asset_request_payload_serialization;
+mod create_detected_asset_response_payload;
+mod create_detected_asset_response_payload_serialization;
+mod create_detected_asset_response_schema;
+mod dataset_target;
+mod default_datasets_destinations_schema_element_schema;
+mod default_events_destinations_schema_element_schema;
+mod default_streams_destinations_schema_element_schema;
+mod destination_configuration;
+mod details_schema_element_schema;
+mod detected_asset;
+mod detected_asset_data_point_schema_element_schema;
+mod detected_asset_dataset_schema_element_schema;
+mod detected_asset_event_schema_element_schema;
+mod detected_asset_response_status_schema;
+mod device;
+mod device_endpoint_schema;
+mod device_inbound_endpoint_schema_map_value_schema;
+mod device_ref_schema;
+mod device_specification_schema;
+mod device_status;
+mod device_status_config_schema;
+mod device_status_endpoint_schema;
+mod device_status_inbound_endpoint_schema_map_value_schema;
+mod device_update_event_schema;
+mod device_update_event_telemetry;
+mod device_update_event_telemetry_receiver;
+mod device_update_event_telemetry_serialization;
+mod event_stream_target;
+mod get_asset_command_invoker;
+mod get_asset_request_payload;
+mod get_asset_request_payload_serialization;
+mod get_asset_response_payload;
+mod get_asset_response_payload_serialization;
+mod get_device_command_invoker;
+mod get_device_response_payload;
+mod get_device_response_payload_serialization;
+mod message_schema_reference;
+mod method_schema;
+mod notification_preference;
+mod notification_preference_response;
+mod qo_s;
```

Comment:
this is weird...is that a codegen thing or a DTDL thing?

Reply:
nothing in this file is written by me